ee/allowedcmd/cmd_darwin.go

```diff
@@ -21,7 +21,7 @@ func Bputil(ctx context.Context, arg ...string) (*exec.Cmd, error) {
 }
 
 func Brew(ctx context.Context, arg ...string) (*exec.Cmd, error) {
-	return validatedCommand(ctx, "/opt/homebrew/bin/brew", arg...)
+	return validatedCommand(ctx, "HOMEBREW_NO_AUTO_UPDATE=1 /opt/homebrew/bin/brew", arg...)
```

Comment:
I'm not sure this will behave correctly. We should, instead, work with the ENV directly.

Reply:
That's the usual way. We could also use `WithAppendEnv` on the wrapper